pkg/operator/sync_openshiftcontrollermanager_v311_00.go

```diff
@@ -38,41 +37,9 @@ func syncOpenShiftControllerManager_v311_00_to_latest(c OpenShiftControllerManag
 	errors := []error{}
 	var err error
 	operatorConfig := originalOperatorConfig.DeepCopy()
-	clientHolder := resourceapply.NewKubeClientHolder(c.kubeClient)
-	directResourceResults := resourceapply.ApplyDirectly(clientHolder, c.recorder, v311_00_assets.Asset,
-		"v3.11.0/openshift-controller-manager/informer-clusterrole.yaml",
-		"v3.11.0/openshift-controller-manager/informer-clusterrolebinding.yaml",
-		"v3.11.0/openshift-controller-manager/ingress-to-route-controller-clusterrole.yaml",
-		"v3.11.0/openshift-controller-manager/ingress-to-route-controller-clusterrolebinding.yaml",
-		"v3.11.0/openshift-controller-manager/tokenreview-clusterrole.yaml",
-		"v3.11.0/openshift-controller-manager/tokenreview-clusterrolebinding.yaml",
-		"v3.11.0/openshift-controller-manager/leader-role.yaml",
-		"v3.11.0/openshift-controller-manager/leader-rolebinding.yaml",
-		"v3.11.0/openshift-controller-manager/ns.yaml",
-		"v3.11.0/openshift-controller-manager/old-leader-role.yaml",
-		"v3.11.0/openshift-controller-manager/old-leader-rolebinding.yaml",
-		"v3.11.0/openshift-controller-manager/separate-sa-role.yaml",
-		"v3.11.0/openshift-controller-manager/separate-sa-rolebinding.yaml",
-		"v3.11.0/openshift-controller-manager/sa.yaml",
-		"v3.11.0/openshift-controller-manager/svc.yaml",
-		"v3.11.0/openshift-controller-manager/servicemonitor-role.yaml",
-		"v3.11.0/openshift-controller-manager/servicemonitor-rolebinding.yaml",
-		"v3.11.0/openshift-controller-manager/buildconfigstatus-clusterrole.yaml",
-		"v3.11.0/openshift-controller-manager/buildconfigstatus-clusterrolebinding.yaml",
-	)
-	resourcesThatForceRedeployment := sets.NewString("v3.11.0/openshift-controller-manager/sa.yaml")
-	forceRollout := false
-
-	for _, currResult := range directResourceResults {
-		if currResult.Error != nil {
-			errors = append(errors, fmt.Errorf("%q (%T): %v", currResult.File, currResult.Type, currResult.Error))
-			continue
-		}
 
-		if currResult.Changed && resourcesThatForceRedeployment.Has(currResult.File) {
-			forceRollout = true
-		}
```

Comment:
@deads2k this was in the original implementation, do we need to check for changes to ocm's ServiceAccount object to force a rollout of the DaemonSet?

Reply:
> @deads2k this was in the original implementation, do we need to check for changes to ocm's ServiceAccount object to force a rollout of the DaemonSet?

I cannot think of a good reason for me doing that.  It is possible that I was concerned about a delete and recreate of the serviceaccount causing new SA tokens to be produced.  I don't think I would worry about it anymore.